infrastructure/iam_roles.tf

```diff
@@ -78,4 +77,120 @@ resource "aws_iam_role_policy" "upload_image_to_dump_s3_cgn_capstone" {
       },
```

Comment:
Is the s3 bucket also part of the terraform? Than maybe you can reference the s3 bucket instead of hard coding it